packages/manager/modules/pci/src/projects/new/payment/components/add/component.js

```diff
@@ -0,0 +1,14 @@
+import controller from './controller';
```

Comment:
Use a component as payment method registration can be done in two situations : if no payment method at all or no default payment method is setted as default.